src/styles/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 10px;
+  width: $cell-size * 7 + $cell-gap * 6;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+
+  $weekdays:
+    mon 0,
+    tue 1,
+    wed 2,
+    thu 3,
+    fri 4,
+    sat 5,
+    sun 6;
+
+  @each $weekday, $value in $weekdays {
+    &--start-day--#{$weekday} &__cell:first-child {
+      margin-left: (($cell-size + $cell-gap) * $value);
+    }
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} &__cell:nth-child(n + #{$day + 1}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    text-align: center;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: background-color 0.5s, transform 0.5s;
```

Comment:
You can organize your css styles in this [order](https://codeguide.academy/html-css.html#css-order)